src/popup/utils/utils.ts

```diff
@@ -597,22 +598,23 @@ export async function readValueFromClipboard(): Promise<string | undefined> {
   if (!process.env.UNFINISHED_FEATURES) {
     return undefined;
   }
-  let value = '';
+  let text = '';
 
   if (IS_CORDOVA) {
-    value = await new Promise((...args) => window.cordova!.plugins!.clipboard.paste(...args));
+    const { type, value } = await Clipboard.read();
+    if (type === 'string') { text = value; }
```

Comment:
```suggestion
    if (type === 'string') { 
      text = value;
    }
```

Otherwise brackets aren't really useful ^^

Reply:
You are right. PR Updated